icu4c/source/test/testdata/te.txt

```diff
@@ -96,7 +96,7 @@ te
     collations {
       standard { 
         Version { "1.0" }
-        Override { "FALSE" }
+        Override { "false" }
```

Comment:
This is not C/C++ code. Safer to leave this one as is.

Reply:
On the contrary, I think there's value in changing to lowercase absolutely everywhere we can, so that no-one ever has to remember that there is some particular corner of ICU where it should be uppercase for no obvious reason.

Reply:
But this is a string value, not code. I haven't looked at the code that parses this string to see if it handles other spellings. Have you?

Reply:
No, I have not looked, I have simply changed them all, that's an advantage too, no need to look.

Reply:
Well, it kind of doesn't make sense, since this is not a C/C++ constant nor talking about one -- but I did look, and this Override value seems to be never read, so it shouldn't matter.